### PR TITLE
add manager provider to experimental settings

### PR DESCRIPTION
**Pull request by valpinkman**
### Type

Feat

### Context

LL-2900

### Parts of the app affected / Test plan

Experimental Settings

CleanShot 2020-07-31 at 14 23 15

### CODE REVIEW

File: src/screens/Settings/Experimental/FeatureInteger.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      let value = v;
+      if (typeof maxValue === "number" && parseInt(value, 10) > maxValue) {
+        value = maxValue;
+      }
+      if (typeof minValue === "number" && parseInt(value, 10) < minValue) {
+        value = minValue;
+      }
+
+      if (!value) {
+        value = maxValue || minValue;
```
Comment (gre) [inline src/screens/Settings/Experimental/FeatureInteger.js:41]: I don't really understand what this is doing. isn't the code above already doing this?

File: src/screens/Settings/Experimental/FeatureRow.js [hunk 2]
```diff
@@ -21,7 +23,7 @@ const FeatureRow = ({ feature }: Props) => {
   return (
     <SettingsRow
       event={`${feature.name}Row`}
-      title={feature.title}
+      title={type === "integer" ? `${feature.title}` : feature.title}
```
Comment (gre) [inline src/screens/Settings/Experimental/FeatureRow.js:26]: mmh why is this needed?

File: src/const/navigation.js [hunk 3]
```diff
@@ -187,6 +187,7 @@ export const NavigatorName = {
   Onboarding: "Onboarding",
   PasswordAddFlow: "PasswordAddFlow",
   PasswordModifyFlow: "PasswordModifyFlow",
+  ManagerProdiver: "ManagerProdiver",
```
[the remark this answers is not in the document]
Comment (juan-cortes) [reply src/const/navigation.js:190]: Unless he's a really good diver

File: src/screens/Settings/Experimental/FeatureRow.js (hunk 2 above)
Comment (juan-cortes) [reply src/screens/Settings/Experimental/FeatureRow.js:26]: Isn't it the same thing?

Comment (valpinkman) [reply src/screens/Settings/Experimental/FeatureRow.js:26]: I wanted to add the current provider into brackets after the title `Provider (5)` but I forgot, maybe not essential so I will remove

File: src/const/navigation.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (valpinkman) [reply src/const/navigation.js:190]: that's some copy pasta from somewhere... so... yeah, will fix

File: src/screens/Settings/Experimental/FeatureInteger.js (hunk 1 above)
Comment (valpinkman) [reply src/screens/Settings/Experimental/FeatureInteger.js:42]: unfortunaly no, when I didn't add that line, I would sometimes end up with a `undefined` inside the input

File: src/const/navigation.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (valpinkman) [reply src/const/navigation.js:190]: actually it's not even used so I remove

**Comment by valpinkman** [comment]
image
latest design

**Comment by valpinkman** [comment]
@Arnaud97234 ready for review

**Comment by valpinkman** [comment]
@Arnaud97234 ready for review, round 3
